### ISSUE TITLE
Lode Runner Level Sequence With CPPNs

### ISSUE THREAD

**Issue by schrum2** [schrum2/MM-NEAT#518]
Once #515 is done, add another task (extending the base level sequence task) that evolves level sequences using CPPNs. The one CPPN input will be the level number, scaled on a range from 0 to 1 (where 1 corresponds to the final level in the sequence).

**Comment by schrum2** [comment]
Evolving level sequences from a single CPPN doesn't actually seem to make that much sense, to this will not be developed further.
